packages/block-library/src/navigation-link/edit.js

```diff
@@ -170,6 +171,14 @@ function NavigationLinkEdit( {
 								className="wp-block-navigation-link__inline-link-input"
 								value={ link }
 								showInitialSuggestions={ true }
+								showCreatePages={ true }
+								createEmptyPage={ ( pageTitle ) =>
+									savePage( {
```

Comment:
Should we set `post_parent` when the added page is a submenu item?

Reply:
So we'd need to grab the attributes of the immediate parent Block, inspect the value of the link and check if it's a Page. If it is then we need to use that as the parent of the created Page.

Given it's complexity would it be fair to make this a phase 2 item? I'm nervous that it will expand the scope quite a bit. What do you think @marekhrabe ?

Reply:
This sounds too big of a feature for the very first iteration. In the last core editor chat, I've learned there is now work being done to allow blocks reading attributes from their parents in a convenient way, I think we should at least wait for that.

Reply:
I would agree. This is after all a Nav and there is nothing to say that a Nav must follow the same hierarchy as defined in `Pages`.

@obenland What say you?

Reply:
Sounds absolutely fair 👍